providers/aws/components/db/setup.ftl

```diff
@@ -505,6 +507,27 @@
                         "reset_master_password || return $?"
                     ],
                 []) +
+                (!rdsCA?has_content)?then(
+                    [
+                        "# Update RDS CA",
+                        "function update_rds_ca() {",
+                        "info \"Updating RDS CA ... \"",
+                        "rdsCAIdentifier=\"rds-ca-2019\"",
```

Comment:
Can we make this a component configuration value and add the script if the saved psedudo stack is different from the one specified in the configuration? 

CA's always have an expiry so it might be nice to make this configurable if we are going to add it